### PR TITLE
Feature/kls 855 856 add legends

### PR DESCRIPTION
**Pull request by davidu1975**
Add legends to search input boxes for

https://www.great.gov.uk/international/investment-support-directory/ and
https://www.great.gov.uk/international/trade

Change styling for these two pages to use either govuk styles or new great styles

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/KLS-855
- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/KLS-856
- [x] Jira ticket has the correct status.
- [x] A clear/description pull request messaged added.

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: core/sass/pages/find-a-supplier/_landing-page.scss [hunk 1]
```diff
@@ -127,7 +97,7 @@ $transparent-white: rgba(250, 250, 250, 0.8);
       display: block;
     }
 
-    .image-container {
+    .ißßß {
```
Comment (lewis-coulson-dit) [inline core/sass/pages/find-a-supplier/_landing-page.scss:100]: guessing this is some rogue key strokes?

File: requirements.in [hunk 2]
```diff
@@ -13,6 +13,7 @@ django-ipware==2.1.0
 django-recaptcha==3.0.0
 django-redis==5.2.*
 django-storages==1.13.1
+<<<<<<< HEAD
```
Comment (lewis-coulson-dit) [inline requirements.in:16]: remove

File: core/sass/pages/find-a-supplier/_landing-page.scss (hunk 1 above)
Comment (davidu1975) [reply core/sass/pages/find-a-supplier/_landing-page.scss:100]: removed

File: requirements.in (hunk 2 above)
Comment (davidu1975) [reply requirements.in]: removed

**Review by lewis-coulson-dit — APPROVED** [review]

**Pull request merged**